### PR TITLE
Add icu plan

### PR DESCRIPTION
**Pull request by stevendanna**
Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @adamhjk to be a potential reviewer

File: plans/icu/plan.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+pkg_origin=core
+pkg_name=icu
+pkg_version=57.1
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("UNICODE, INC. LICENSE AGREEMENT")
```
Comment (metadave) [inline plans/icu/plan.sh]: Is this the same as http://spdx.org/licenses/Unicode-TOU.html ?

**Comment by metadave** [comment]
looks good other than the license question

**Comment by stevendanna** [comment]
Updated the license info

**Comment by metadave** [comment]
@thesentinels delegate=stevendanna

**Comment by thesentinels** [comment]
:v: @stevendanna can now approve this pull request

**Comment by thesentinels** [comment]
:v: @stevendanna can now approve this pull request

**Comment by stevendanna** [comment]
@thesentinels r+

**Comment by thesentinels** [comment]
:pushpin: Commit 52144df has been approved by `stevendanna`

**Comment by thesentinels** [comment]
:hourglass: Testing commit 52144df with merge 9ee8948...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/habitat-sh/habitat/builds/139749765)

**Comment by metadave** [comment]
built and uploaded: https://app.habitat.sh/#/pkgs/core/icu/57.1/20160622204012

**Pull request merged**